Documentation/debugging/using-your-build.md

```diff
@@ -0,0 +1,70 @@
+Using Your Build
```

Comment:
I'm not sure if the `building/` folder would have been a better place for this. I thought of this as more debugging because when I want to use my build in a console app I usually want to debug it, but after writing the document I'm not sure.